frontend/src/metabase/public/components/EmbedModal/StaticEmbedSetupPane/lib/get-has-settings-changes.ts

```diff
@@ -0,0 +1,34 @@
+import _ from "underscore";
+
+import type { EmbeddingParameters } from "metabase/public/lib/types";
+
+function getNonDisabledEmbeddingParams(
+  embeddingParams: EmbeddingParameters,
+): EmbeddingParameters {
+  return Object.keys(embeddingParams).reduce((result, key) => {
+    if (embeddingParams[key] !== "disabled") {
+      result[key] = embeddingParams[key];
+    }
+
+    return result;
+  }, {} as EmbeddingParameters);
+}
+
+export function getHasSettingsChanges({
```

Comment:
Looks like this is only for embedding parameters, not settings?

```suggestion
export function getHasParamsChanged({
```